DxDiagram: default item properties (#700)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/defaultItemProperties/style.md

---
---
##### shortDescription
Specifies a default item style.

---

> The **style** option specifies a default style that is applied to a shape/connector if there is not another value provided via the [nodes.styleExpr](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#styleExpr)/[edges.styleExpr](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/edges/#styleExpr) option.

Reviewer: ```suggestion
The **style** option specifies the default style applied to a shape/connector if the [nodes.styleExpr](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#styleExpr)/[edges.styleExpr](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/edges/#styleExpr) option does not provide another value. 
```
